deepchem/utils/dft_utils/config.py

```diff
@@ -0,0 +1,37 @@
+"""
+Density Function Theory Configuration Utilities.
+"""
+
+from dataclasses import dataclass
+
+__all__ = ["config"]
+
+
+@dataclass
+class _Config(object):
+    """Contains the configuration for the DFT module
+
+    Examples
+    --------
+    >>> from deepchem.utils.dft_utils.config import config
+    >>> Memory_usage = 1024**4 # Sample Memory usage by some Object/Matrix
+    >>> if Memory_usage > config.THRESHOLD_MEMORY :
+    ...     print("Overload")
+    Overload
+
+    Attributes
+    ----------
+    THRESHOLD_MEMORY: int
+        Threshold memory (matrix above this size should not be constructed)
+    CHUNK_MEMORY: int
+        The memory for splitting big tensors into chunks
+    VERBOSE: int
+        Verbosity level
```

Comment:
Need more clarification on the attributes and how they help in context to the task you are gonna perform. (like verbosity level can be explained further)